app/models/user.rb

```diff
@@ -17,6 +18,37 @@ class User < ApplicationRecord
             length: {minimum: Settings.digit.length_6}
   has_secure_password
 
+  class << self
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost:
+    end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
ri là hợp lý nhất bé nha